public/v1/apps/drone.json

```diff
@@ -0,0 +1,102 @@
+{
+    "captainVersion": "1",
+    "documentation": "Taken from https://hub.docker.com/r/drone/drone/",
+    "dockerCompose": {
+        "version": "2",
+        "services": {
+            "$$cap_appname-agent": {
+                "image": "drone/agent:$$cap_drone_version",
+                "notExposeAsWebApp": "true",
+                "restart": "always",
+                "environment": {
+                    "DRONE_SERVER": "srv-captain--$$cap_appname-server:9000",
+                    "DRONE_SECRET": "$$cap_drone_secret"
+                }
+            },
+            "$$cap_appname-server": {
+                "depends_on": [
+                    "$$cap_appname-agent"
+                ],
+                "image": "drone/drone:$$cap_drone_version",
+                "containerHttpPort": "80",
+                "volumes": [
+                    "/var/run/docker.sock:/var/run/docker.sock",
+                    "$$cap_appname-server:/data"
+
+                ],
+                "restart": "always",
+                "environment": {
+                    "DRONE_GITEA_SERVER": "$$cap_drone_gitea_server",
+                    "DRONE_GIT_ALWAYS_AUTH": "$$cap_drone_gitea_always_auth",
+                    "DRONE_RUNNER_CAPACITY": "$$cap_drone_runner_capacity",
+                    "DRONE_SERVER_PROTO": "$$cap_drone_server_proto",
+                    "DRONE_SERVER_HOST": "$$cap_drone_server_host",
+                    "DRONE_TLS_AUTOCERT": "$$cap_drone_tls_autocert",
+                    "DRONE_SECRET": "$$cap_drone_secret"
+                }
+            }
+        }
+    },
+    "instructions": {
+        "start": "Drone is a self-service Continuous Delivery platform for busy development teams.\n\n Enter your Drone for Gitea configuration parameters and click on next. The process will take about a minute to finish.",
```

Comment:
It seems like Drone can work with many git clients (including Gitea). But this drone config is geared only towards Gitea.

I might be wrong. But if my assumption above is correct, let's:
- rename this file to `drone-gitea` instead of `drone` 
- Add clear instructions in `instructions.start` saying that you need to have an already running Gitea server. This does not automatically spin up a Gitea instance for you.

Reply:
agreed